deploy/Dockerfile

```diff
@@ -6,6 +6,9 @@ WORKDIR /app
 COPY . .
 # lint check
 RUN flake8 --statistics --verbose
+# prepare local dev environment, for tests execution
+RUN pip install -e .
```

Comment:
We should probably do this in the base image no ?

Reply:
we can't, it depends on setup.py which depends on more source code: we don't want that in the base image